tests/unit/pypyr/dsl_test.py

```diff
@@ -2211,6 +2234,37 @@ def test_run_pipeline_steps_complex_with_skip_true(mock_invoke_step,
     assert len(context) == original_len
 
 
+@ patch('pypyr.moduleloader.get_module')
```

Comment:
@yaythomas what is a preferred way to write a decorator?

with space or without, ex. `@ patch` or `@patch`. I saw both styles at the tests

Reply:
oh, hell - this is an argument I've got going with my IDE. . . sometimes on auto-save/auto-format it puts spaces in between the @ and the prop name. Whether this is a bug or a setting I've not found, I don't know. So sometimes `@ patch` sneaks in when I don't catch the times (not always) it does that.

Which is to say in a perfect world, I generally prefer no space `@patch`.